test/validate_test.js

```diff
@@ -90,6 +90,29 @@ suite('api/validate', function() {
   }, function(req, res) {
     res.reply({value: 'Hello World'});
   });
+  
+  // Declare a method we can use to test res.reply with empty body
+  builder.declare({
+    method:   'get',
+    route:    '/test-res-reply',
+    name:     'testResReplyGet',
+    output:   'test-schema.yml',
```

Comment:
The test that uses this method should fail, because an empty body does not match schema `test-schema.yml`.  So I'd like to see a change such that `validate.js` will return a failure status code in this case, and then the tests adjusted to match.

Reply:
The tests are passing with an empty response body `res.reply()` but when I am changing it to `res.reply({})` then it's failing. 
Do you want something like this in the test?

Reply:
This is a case where both the tests and the code are wrong.  With the api method definition here, `res.reply()` should fail.